src/smart-components/group/principal/principals.js

```diff
@@ -14,7 +14,7 @@ import AddGroupMembers from './add-group-members';
 import { Section } from '@redhat-cloud-services/frontend-components';
 import RemoveModal from '../../../presentational-components/shared/RemoveModal';
 
-const columns = [{ title: 'Name', cellFormatters: [ expandable ]}, 'Email', 'First name', 'Last name' ];
+const columns = [{ title: 'Username', cellFormatters: [ expandable ]}, 'Email', 'First name', 'Last name' ];
```

Comment:
No need to change this here since this PR is about wizard columns and such. As mentioned before this is being addressed in https://github.com/RedHatInsights/insights-rbac-ui/pull/166

Reply:
Ah, https://projects.engineering.redhat.com/browse/RHCLOUD-4502 talked about columns (not just in wizards), so I was just fixing that too. I'll remove it since it's being addressed in other places.